target/scripts/startup/setup.d/security/rspamd.sh

```diff
@@ -31,33 +30,55 @@ function _setup_rspamd
 # @param ${2} = message
 function __rspamd__log { _log "${1:-}" "(Rspamd setup) ${2:-}" ; }
 
-# Run miscellaneous checks against the current configuration so we can
-# properly handle integration into ClamAV, etc.
+# Helper for explicitly enabling or disabling a specific module.
 #
-# This will also check whether Amavis is enabled and emit a warning as
-# we discourage users from running Amavis & Rspamd at the same time.
-function __rspamd__preflight_checks_and_setup
+# @param ${1} = module name
+# @param ${2} = `true` when you want to enable the module (default),
+#               `false` when you want to disable the module [OPTIONAL]
+# @param ${3} = whether to use `local` (default) or `override` [OPTIONAL]
+function __rspamd__helper__enable_disable_module
 {
-  touch /var/lib/rspamd/stats.ucl
+  local MODULE=${1:?Module name must be provided}
+  local ENABLE_MODULE=${2:-true}
+  local LOCAL_OR_OVERRIDE=${3:-local}
+  local MESSAGE='Enabling'
 
-  if [[ ${ENABLE_AMAVIS} -eq 1 ]] || [[ ${ENABLE_SPAMASSASSIN} -eq 1 ]]
+  if [[ ! ${ENABLE_MODULE} =~ ^(true|false)$ ]]
   then
-    __rspamd__log 'warn' 'Running Amavis/SA & Rspamd at the same time is discouraged'
+    __rspamd__log 'warn' "__rspamd__helper__enable_disable_module got non-boolean argument for deciding whether module should be enabled or not"
+    return 1
   fi
 
-  if [[ ${ENABLE_CLAMAV} -eq 1 ]]
+  [[ ${ENABLE_MODULE} == true ]] || MESSAGE='Disabling'
+
+  __rspamd__log 'trace' "${MESSAGE} module '${MODULE}'"
+  cat >"/etc/rspamd/${LOCAL_OR_OVERRIDE}.d/${MODULE}.conf" << EOF
+# documentation: https://rspamd.com/doc/modules/${MODULE}.html
+
+enabled = ${ENABLE_MODULE};
+
+EOF
+}
+
+# Run miscellaneous early setup tasks and checks, such as creating files needed at runtime
+# or checking for other anti-spam/anti-virus software.
+function __rspamd__run_early_setup_and_checks
+{
+  touch /var/lib/rspamd/stats.ucl
```

Comment:
In other places we use this:
```suggestion
  : >/var/lib/rspamd/stats.ucl
```

Reply:
Done in [1ecdfe6](https://github.com/docker-mailserver/docker-mailserver/pull/3206/commits/1ecdfe60abc48ec11ef1238455aa7bac686dec14)